css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-avatar-dimensions: 16 24 32 36 48 !default;
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-default-size: 32px !default;
+$persona-font-size: $font-size-7 !default;
+$persona-gap-size: $layout-1 !default;
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+$persona-dimension-font-size-ratio: 0.45;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+	font-size: $persona-font-size;
+
+	.persona-avatar {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		width: $persona-default-size;
+		height: $persona-default-size;
+		border-radius: $border-radius-rounded;
+		background-color: $persona-avatar-background-color;
+		color: $persona-avatar-font-color;
+		font-size: $persona-default-size * $persona-dimension-font-size-ratio;
+		overflow: hidden;
+
+		img,
+		svg {
+			display: block;
+			width: 100%;
+			height: auto;
+			max-height: 100%;
+		}
+
+		span {
+			line-height: 1;
+			text-transform: uppercase;
+		}
+	}
+
+	.persona-description :not(:first-child) {
```

Comment:
Can you explain why `:not(:first-child)` is needed here for me?

Reply:
The idea behind it is that the first child will be the user's name and everything after it will be metadata with a bit different styling - font size is smaller and has a bit lighter text color. So I thought it's easier to target it this way rather than creating another class.

Reply:
Refactored this part